content/applications/hr/lunch/orders.rst

```diff
@@ -0,0 +1,210 @@
+order is visible. The default filter for the products is :guilabel:`Available Today`. This filter
+shows only products that can be purchased that day.
+
+The left-side of the dashboard displays the various :guilabel:`Categories` of products available,
+along with the :guilabel:`Vendors` supplying the products. On the far-right side of each line is a
+number, which indicates how many products are associated with that respective category or vendor.
+
+To filter the products by categories or vendors, click the checkbox next to the specific category or
+vendor to view only those items.
+
+.. note::
+   If multiple selections are made, **only** products that fall under **all** the selected options
+   are shown.
+
+The top portion of the dashboard, the :guilabel:`Your Order` summary, displays the user's account
+information, and the order details for the current day, if any orders have been placed.
+
+The main section, beneath the user's information, houses all the products in a Kanban view. Each
+product card displays the name, cost, vendor, photo, and a description of the product. If the
+product is configured as new, it also displays a :guilabel:`New` tag.
+
+.. image:: orders/dashboard.png
+   :align: center
+   :alt: The *Lunch* app dashboard with all areas highlighted.
+
+.. note::
+   Anywhere the vendor is listed in the *Lunch* app, such as the Kanban product cards, both the name
+   of the vendor appears as well as their phone number.
+
+The products can be displayed in a list view by clicking the :guilabel:`≣ (four parallel lines)`
+icon in the top-right.
+
+Placing orders
+==============
+
+To place a lunch order, navigate to the main :guilabel:`Order Your Lunch` dashboard.
+
+Add products to an order
+------------------------
+
+Click on a product to order, and the product appears in a :guilabel:`Configure Your Order` pop-up
+window.
+
+At the top of the pop-up window is the product image, name, and price. Beneath that are any
+available :ref:`extra items or options <lunch/extra>`. Click the checkbox next to any extras to add
+them to the order.
+
+Each extra is organized by a category, and lists the name and price for each extra. As extras are
+selected, the displayed price at the top of the pop-up window updates to reflect all current
+selections.
+
+Beneath the extras is the :guilabel:`Description` of the product, followed by a :guilabel:`Notes`
+field. The :guilabel:`Notes` field is used to enter any information to be sent to the vendor
+regarding the order, such as any special requests or food allergies.
+
+When all selections for the product have been made, click the :guilabel:`Add To Cart` button in the
+lower-left of the pop-up window. To cancel the order, click the :guilabel:`Discard` button.
+
+.. image:: orders/pizza.png
+   :align: center
+   :alt: The pop-up window for a personal pizza, with all the extras highlighted and selected.
+
+Errors
+~~~~~~
```

Comment:
Maybe put this section last? We shouldn't front load with the flaws of the app. Generally I add a troubleshooting section last in docs. This makes a potential customer think that the app doesn't work properly at first glance.

Just a suggestion, up to you!

Reply:
Is this expected behavior?  How does the customer know this before they get the error? Is there a warning?